content/config_rb_client.md

```diff
@@ -430,9 +432,9 @@ This configuration file has the following settings:
 `log_level`
 
 :   The level of logging to be stored in a log file. Possible levels:
-    `:auto` (default), `:trace`(starting in 14.0.202), `:debug`, `:info`, `:warn`, `:error`,
-    or `:fatal`. Default value: `:warn` (when a terminal is available)
-    or `:info` (when a terminal is not available).
+    `:auto` (default), `:trace`, `:debug`, `:info`, `:warn`, `:error`,
+    or `:fatal`. The `:auto` level will use `:warn` when a terminal is
+    available or `:info` when a terminal is not available.
```

Comment:
This may be wrong now, I think we use `:info` by default and `:warn` is only used now when switching to syslog-ing, or when the user explicitly does `--force-logger`.

Reply:
(honestly I'm not sure how to document this right now for Chef-16, that's the best I can come up, I would need to burn a lot of time testing it, and I wouldn't know how to English that up for user consumption since it is a pile of edge conditions nearly nobody should care about any more)

Reply:
@lamont-granquist Wanna toss a suggestion to change that?

Reply:
From Slack "Just ship it"